modules/caddyhttp/reverseproxy/fastcgi/client_test.go

```diff
@@ -159,11 +161,12 @@ func sendFcgi(reqType int, fcgiParams map[string]string, data []byte, posts map[
 			length += len(k) + int(fi.Size())
 		}
 		resp, err = fcgi.PostFile(fcgiParams, values, files)
+		resp.Body.Close()
```

Comment:
You might want to wait until the `Body` is read below on line 170 before closing it, no ?